### PR TITLE
Collect telemetry

### PR DESCRIPTION
**Pull request by ricklue**
Created an API to gather telemetry data from OpenTelemetry (https://opentelemetry.io/). When code is instrumented with the OpenTelemetry library, it is possible to use the endpoint from this API to gather it in Ladybug.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -52,6 +52,18 @@
     </developer>
   </developers>
 
+  <dependencyManagement>
```
Comment (jacodg) [inline pom.xml:55]: Why are you adding `<dependencyManagement>`? Can you remove it and only use `<dependencies>`?

Comment (ricklue) [reply pom.xml]: Added the change

**Comment by jacodg** [comment]
Can you add info to the README about Ladybug being able to receive OpenTelemetry data? And maybe an example of how to send information to Ladybug using the new API endpoint?

File: src/main/java/nl/nn/testtool/web/ApiAuthorizationFilter.java [hunk 2]
```diff
@@ -91,6 +91,7 @@ public void setObserverRoles(List<String> observerRoles) {
 		addConfigurationPart("PUT/"  + ApiServlet.LADYBUG_API_PATH + "/testtool/node-link-strategy$", observerRoles);
 		addConfigurationPart("GET/"  + ApiServlet.LADYBUG_API_PATH + "/metadata/.*$", observerRoles);
 		addConfigurationPart("GET/"  + ApiServlet.LADYBUG_API_PATH + "/report/.*$", observerRoles);
+		addConfigurationPart("POST/" + ApiServlet.LADYBUG_API_PATH + "/collector/.*$", observerRoles);
```
Comment (jacodg) [inline src/main/java/nl/nn/testtool/web/ApiAuthorizationFilter.java:94]: I think we need to create a separate role. Can you remove this line and add the following below the setTesterRoles() method?

	public void setWebServiceRoles(List<String> webServiceRoles) {
		if (constructorDone) log.info("Set web service roles");
		addConfigurationPart("POST/" + ApiServlet.LADYBUG_API_PATH + "/collector/.*$", webServiceRoles);
	}

And add setWebServiceRoles(null); below setTesterRoles(null);?

**Review by jacodg — CHANGES_REQUESTED** [review]

File: src/main/java/nl/nn/testtool/web/ApiAuthorizationFilter.java (hunk 2 above)
Comment (ricklue) [reply src/main/java/nl/nn/testtool/web/ApiAuthorizationFilter.java]: Added the new role

**Pull request merged**